common/static/common/js/karma.common.conf.js

```diff
@@ -306,8 +296,6 @@ function getBaseConfig(config, useRequireJs) {
             'karma-requirejs',
             'karma-junit-reporter',
             'karma-coverage',
-            'karma-chrome-launcher',
```

Comment:
As we have decided we will not remove the `karma` test suite-related things, do these configs needs to be removed anyways?